client/src/pages/LandingPage/LandingPage.js

```diff
@@ -1,24 +1,134 @@
+                                                    name='username'
+                                                    onChange={this.handleInputChange}
+                                                />
+                                            </Col>
+                                        </div>
+                                    </FormGroup>
+                                    <FormGroup>
+                                        <Row>
+                                            <Col xs={12} md={6}>
+                                                <Button className='landingBtn' bsSize='large' bsStyle='primary' type="submit" onClick={this.handleSubmit} block>Log in</Button>
+                                            </Col>
+                                            <Col xs={12} md={6}>
+                                                <Button className='landingBtn' bsSize='large' type="create" block href='/register'>Create Account</Button>
+                                            </Col>
+                                        </Row>
+                                    </FormGroup>
+                                </Wrapper>
+                            </Form>
+                        </Well>
+                    </Wrapper>
+                </div>
+                <div>
+                    <Wrapper>
+                        <Row>
+                            <h2 className='landingSubTitle'>Connect with Other Brewers</h2>
+                            <hr className='landingHR' />
+                        </Row>
+                        <Row>
+                            <Col xs={6}>
+                                <div className='test' />
+                                {/* <Image src='../../images/alexander-mils-431544-unsplash.jpg' alt='glass of beer' rounded responsive /> */}
+                            </Col>
+                            <Col xs={6}>
+                                <Row>
+                                    <Col xs={12}>
+                                        <ul className='landingList'>
+                                            <li>Learn how to brew from others</li>
+                                            <li>Share your recipes and get feedback</li>
+                                            <li>Connect with like minded people</li>
+                                            <li>Save your own recipes and collect your favorites all in one place</li>
+                                        </ul>
+                                    </Col>
+                                    <Col xs={12}>
+                                        <Row>
+                                            <Col xs={0} md={4} />
+                                            <Col xs={12} md={4}>
+                                                <Button className='landingBtn' bsSize='large' block href='/register'>Sign up</Button>
+                                            </Col>
```

Comment:
We may want to break the login part into another component. It seems to defeat the purpose of using React by having these sections jammed into one page instead of broken down into different components.